File: Python/src/gestorAplicacion/empleado.py
from gestorAplicacion.persona import Persona

class Empleado(Persona):

    def __init__(self, cedula, nombre, telefono, direccion, cargo, sueldo):
        super().__init__(cedula, nombre, telefono, direccion)
        self._cargo=cargo
        self._sueldo=sueldo

 #getters y setters
    #def setId(self, id):
    #     self.id = id

    #def getId(self):
    #     return self.id    
    
    def setCargo(self, cargos):
         self._cargo = cargos

    def getCargo(self):
        return self._cargo 

    def setSueldo(self,sueldos):
        self._sueldo = sueldos

    def getSueldo(self):
        return self._sueldo     

File: src/gestorAplicacion/Empleado.java
package gestorAplicacion;

public class Empleado extends Persona{
	private int id;
	private String cargo;
	private int sueldo;
	
	public Empleado(int cedula, String nombre, int telefono, String direccion, int id, String cargo, int sueldo) {
		super(cedula, nombre, telefono, direccion);
		this.setId(id);
		this.setCargo(cargo);
		this.sueldo = sueldo;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getCargo() {
		return cargo;
	}

	public void setCargo(String cargo) {
		this.cargo = cargo;
	}

	public void setSueldo(int sueldo){ this.sueldo = sueldo;}

	public int getSueldo(){return this.sueldo;}

}
